# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1260343230 18000
#      Wed Dec 09 02:20:30 2009 -0500
# Node ID 31436dcf7d0fc2d905205076b0aaa0c08319dbcb
# Parent  55061c7c8d6edd245848603db474b38cc92b81b9
call nstrftime, not my_strftime

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-09  John W. Eaton  <jwe@octave.org>
+
+	* oct-time.cc (octave_base_tm::strftime): Call nstrftime instead
+	of my_strftime.
+
 2009-12-09  John W. Eaton  <jwe@octave.org>
 
 	* oct-time.cc: Include strftime.h.  Don't provide decl for strftime.
 	(octave_base_tm::strftime): Call my_strftime instead of ::strftime.
 	* Makefile.am (AM_CPPFLAGS): Add -I$(top_srcdir)/libgnu to the list.
 	(LINK_DEPS): Add ../libgnu/libgnu.la to the list.
 	(LIBOCTAVE_C_SOURCES): Remove strftime.c from the list.
 	* strftime.c: Delete.
diff --git a/liboctave/oct-time.cc b/liboctave/oct-time.cc
--- a/liboctave/oct-time.cc
+++ b/liboctave/oct-time.cc
@@ -262,17 +262,17 @@ octave_base_tm::strftime (const std::str
       size_t chars_written = 0;
 
       while (chars_written == 0)
 	{
 	  delete [] buf;
 	  buf = new char[bufsize];
 	  buf[0] = '\0';
 
-	  chars_written = my_strftime (buf, bufsize, fmt_str, &t, 0, 0);
+	  chars_written = nstrftime (buf, bufsize, fmt_str, &t, 0, 0);
 
 	  bufsize *= 2;
 	}
 
 #if defined (HAVE_STRUCT_TM_TM_ZONE)
       delete [] ps;
 #endif
 
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-12-09  John W. Eaton  <jwe@octave.org>
+
+	* load-save.cc: Call nstrftime instead of my_strftime.
+
 2009-12-09  John W. Eaton  <jwe@octave.org>
 
 	* load-save.cc: Include strftime.h.
 	(write_header): Call my_strftime instead of strftime.
 
 	* Makefile.am (AM_CPPFLAGS): Add -I$(top_srcdir)/libgnu to the list.
 	(OCTAVE_LIBS, OCTINTERP_LINK_DEPS, OCT_LINK_DEPS):
 	Add ../libgnu/libgnu.la to the list.
diff --git a/src/load-save.cc b/src/load-save.cc
--- a/src/load-save.cc
+++ b/src/load-save.cc
@@ -1179,18 +1179,18 @@ write_header (std::ostream& os, load_sav
 	struct tm bdt;
 	time_t now;
 	char headertext[128];
 
 	time (&now);
 	bdt = *gmtime (&now);
 	memset (headertext, ' ', 124);
 	// ISO 8601 format date
-	my_strftime (headertext, 124, "MATLAB 5.0 MAT-file, written by Octave "
-		     OCTAVE_VERSION ", %Y-%m-%d %T UTC", &bdt, 1, 0);
+	nstrftime (headertext, 124, "MATLAB 5.0 MAT-file, written by Octave "
+                   OCTAVE_VERSION ", %Y-%m-%d %T UTC", &bdt, 1, 0);
 
 	// The first pair of bytes give the version of the MAT file
 	// format.  The second pair of bytes form a magic number which
 	// signals a MAT file.  MAT file data are always written in
 	// native byte order.  The order of the bytes in the second
 	// pair indicates whether the file was written by a big- or
 	// little-endian machine.  However, the version number is
 	// written in the *opposite* byte order from everything else!
